<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GameDialog</class>
 <widget class="QDialog" name="GameDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1500</width>
    <height>987</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QScrollArea" name="scrollArea">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>40</y>
     <width>700</width>
     <height>700</height>
    </rect>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>698</width>
      <height>698</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QWidget" name="wdg_qipan" native="true"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QScrollArea" name="scrollArea_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>40</y>
     <width>351</width>
     <height>450</height>
    </rect>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>349</width>
      <height>448</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QWidget" name="wdg_player1" native="true"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QScrollArea" name="scrollArea_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>520</y>
     <width>351</width>
     <height>450</height>
    </rect>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents_3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>349</width>
      <height>448</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QWidget" name="wdg_player2" native="true"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QPushButton" name="pb_startGame">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>810</y>
     <width>511</width>
     <height>121</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>40</pointsize>
    </font>
   </property>
   <property name="text">
    <string>开始游戏</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
